src/server/pkg/hashtree/db.go

```diff
@@ -265,6 +265,10 @@ func List(rs []io.ReadCloser, pattern string, f func(string, *NodeProto) error)
 func glob(tx *bolt.Tx, pattern string, f func(string, *NodeProto) error) error {
 	if !IsGlob(pattern) {
 		node, err := get(tx, pattern)
+		if Code(err) == PathNotFound {
+			// glob pattern is for a static file that doesn't exist, no match
```

Comment:
I don't think it really makes sense to call a file in pfs static vs non-static because there is no notion of a non-static file in pfs.

Reply:
Done.